projects/distributed-tracing/src/shared/dashboard/interaction/detail-sheet/detail-sheet-interaction-handler.model.ts

```diff
@@ -18,30 +18,36 @@ export class DetailSheetInteractionHandlerModel implements InteractionHandler {
 
   @ModelProperty({
     key: 'sheet-size',
-    required: true,
+    required: false,
     // tslint:disable-next-line: no-object-literal-type-assertion
     type: {
       key: ENUM_TYPE.type,
       values: [SheetSize.Small, SheetSize.Medium, SheetSize.Large]
     } as EnumPropertyTypeInstance
   })
-  public sheetSize!: SheetSize;
+  public sheetSize: SheetSize = SheetSize.Large;
+
+  @ModelProperty({
+    key: 'inject-source-as',
+    type: STRING_PROPERTY.type
+  })
+  public injectSourceAs: string = 'source';
 
   @ModelInject(MODEL_API)
   private readonly api!: ModelApi;
 
   @ModelInject(DetailSheetInteractionHandlerService)
   private readonly handlerService!: DetailSheetInteractionHandlerService;
 
-  public execute(record: unknown): Observable<void> {
-    this.handlerService.showSheet(this.getDetailModel(record), this.sheetSize);
+  public execute(source: unknown): Observable<void> {
+    this.handlerService.showSheet(this.getDetailModel(source), this.sheetSize);
 
     return of();
   }
 
-  private getDetailModel(record: unknown): object {
+  private getDetailModel(source: unknown): object {
```

Comment:
createChild will default to this model as the parent

Reply:
removed